### ISSUE TITLE
Wrong typings

### ISSUE THREAD

**Issue by Marsup** [fastify/fastify-multipart#76]
The types of this module specify there's a default export (https://github.com/fastify/fastify-multipart/blob/master/index.d.ts#L81) while the export is a commonjs one (https://github.com/fastify/fastify-multipart/blob/master/index.js#L174).

It seems hard to fix in these typings since there are some other stuff exported, and I see `fastify-plugin` kind of tries to do things with the default export, so I'm wondering should it be fixed here or there ?

**Comment by Eomm** [comment]
Do you have some specific error?
How would you fix this?

Because all the typing of the fastify plugins are configured like this one (an external `index.d.ts` file and the `index.js` that is a node.js module, put together with the package.json using the `types` field )

**Comment by Marsup** [comment]
Well there are 2 ways to import considering the definition:
1) `import * as fastifyMultipart from 'fastify-multipart';`
2) `import fastifyMultipart from 'fastify-multipart';`

The 1st one is (surpisingly) the correct one, the 2nd one will be `undefined`, because there's no `default` property, since it's commonjs. That is probably unless you use typescript with `esModuleInterop` which we don't.

If I go with 1), then fastify typings are unhappy:
```
Error:(14, 20) TS2345: Argument of type 'typeof import("./node_modules/fastify-multipart/index")' is not assignable to parameter of type 'Plugin<Server, IncomingMessage, ServerResponse, RegisterOptions<Server, IncomingMessage, ServerResponse>>'.
  Type 'typeof import("./node_modules/fastify-multipart/index")' provides no match for the signature '(instance: FastifyInstance<Server, IncomingMessage, ServerResponse>, opts: RegisterOptions<Server, IncomingMessage, ServerResponse>, callback: (err?: FastifyError | undefined) => void): void'.
```
If I go with 2), there's no type error but obviously fastify fails at runtime telling me plugins should be functions.

**Comment by mcollina** [comment]
Would you mind sending a PR to update the typings?

**Comment by Marsup** [comment]
Updating the types doesn't seem possible at first glance as you can't have both normal exports and an `export = ...`, you could keep the types as is if `fastify-plugin` created a `default` property when none exists. Would that be an acceptable fix ?

**Comment by mcollina** [comment]
I think we should do:

```
export = fastifyMultipart;
```

**Comment by Eomm** [comment]
Fixed in #78 and released in 1.0.1 👍
